File: Ejercicio/GO/MetodoConParametro/MetodoCalcular.go
package main
import "fmt"

func areaRectangulo(base float64, altura float64) float64 {
    return base * altura
}

func main() {
    area := areaRectangulo(5.5, 3.2)
    fmt.Println("Área del rectángulo:", area)
}

File: Ejercicio/Java/MetodoConParametro/MetodoCalcular.java
public class MetodoCalcular {
    static double areaRectangulo(double base, double altura) {
        return base * altura;
    }

    public static void main(String[] args) {
        double area = areaRectangulo(5.5, 3.2);
        System.out.println("El área del rectángulo es: " + area);
    }
}
